code/midtown/mmaudio/sound.h

```diff
@@ -91,6 +91,8 @@
 
 #include "arts7/node.h"
 
+#include "mmai/aiGoalFollowWayPts.h"
```

Comment:
Why is this included here?

Reply:
If omitted we will get "incomplete type is not allowed" for:
    Vector3 LastPosition;
    Vector3 Velocity;

Reply:
Then you need to include the header for Vector3 (`vector7/vector3.h`)